### ISSUE TITLE
Landing files

### ISSUE THREAD

**Issue by QuentinG50** [Estia-advanced-programming/pandora-public#33]
Should we use the files 201-Mig ou 301-Mig to calculate the landing ? because the 201 reffer to weird values at the end of the file.

**Comment by WilliamDelamare** [comment]
Sorry for the late reply, I missed that question.

Quick answer: you should consider all kind of files!
Long answer: yes, the flights are all different (some jets landed higher than 2000m, in the mountains). If you follow the `yaw` solution described on the wiki, it should provide you a reasonable approximation of how the flights happened.

**Comment by QuentinG50** [comment]
ok we go it, 
But we finally find errors between our values which are enormous and the ones coutains in the .jar files. We supposed the origin coming from the distance calcul that you validate me on friday coming (write on the website with radians) and impact all our value using (distance, groundspeed, acce) by a step of 2-2.5 time higher values.

**Comment by WilliamDelamare** [comment]
Yes, you and another group highlighted a bug (https://github.com/Estia-advanced-programming/pandora-public/issues/34#issuecomment-600669493)

(should be corrected  in the following release)
